administrator/modules/mod_user/tmpl/default.php

```diff
@@ -24,7 +25,7 @@
 			<span class="fa fa-user-circle" aria-hidden="true"></span>
 		</div>
 		<div class="tiny">
-			<?php echo Text::_('MOD_USER_MENU'); ?>
+			<?php echo htmlspecialchars(Factory::getUser()->name); ?>
```

Comment:
```suggestion
			<?php echo htmlspecialchars($app->getIdentity()->name); ?>
```
And remove `use Joomla\CMS\Factory;`.

Reply:
ty